### ISSUE TITLE
Please tell users what Github PAT scopes you need

### ISSUE THREAD

**Issue by TylerR909** [WowUp/WowUp#1256]
**Is your feature request related to a problem? Please describe.**
I'm adding a Github PAT to WowUp right now and I don't want to give you God Powers over my repos. 

**Describe the solution you'd like**
Please tell users the list of permissions/scopes they need, directly on the Addons page, instead of making me guess.

**Comment by jliddev** [comment]
I do not believe you need to add any permissions to the base PAT for downloads to work. Adding a section for this to the guide is a great idea.
